### DISCUSSION TITLE
Is there 'disable all' option in config?

### DISCUSSION THREAD

**Discussion by tetafro** [starship/starship#1988] [Q&A] [answered]
I want to disable all modules except those that I explicitly enabled. Can I do it with a single command, and not by disabling every single module in config?

**Comment by davidkna** [comment, marked as answer]
You could use the top-level [`format`](https://starship.rs/config/#prompt) option and only list the modules you want enabled there.

**Reply by tetafro** [reply]
Right. Thanks.
There're even issues on that. Don't know how I missed them
- https://github.com/starship/starship/issues/1973
- https://github.com/starship/starship/issues/949
